puppet/puppet-bootstrap.sh

```diff
@@ -1,49 +1,27 @@
 #!/bin/bash -e
 
-PUPPET_VERSION=${1:-nothing}
```

Comment:
Correct. It's a massive pain on the puppet server side, especially if and when we switch to hiera for this stuff.